oxide-auth/src/primitives/registrar.rs

```diff
@@ -120,10 +121,10 @@ pub struct ExactUrl(String);
 /// considered.
 ///
 /// [`ExactUrl`]: ExactUrl
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 pub struct IgnoreLocalPortUrl(IgnoreLocalPortUrlInternal);
```

Comment:
Similar issue here as for `ExactUrl` but here it's actually _safety relevant_ that the `Local`-variant must not be constructed from a non-`localhost` domain. Deserialization _must_ critically go through `IgnoreLocalPortUrl::new` for this struct.